beluga_amcl/test/test_amcl_node.cpp

```diff
@@ -473,7 +476,7 @@ TEST_F(TestNode, SetInitialPose) {
   amcl_node_->activate();
   tester_node_->publish_map();
   ASSERT_TRUE(wait_for_initialization());
-  const auto [pose, _] = amcl_node_->particle_filter()->estimate();
+  const auto [pose, _] = amcl_node_->estimate();  // <-- bad optional access
```

Comment:
@nahueespinosa hmm, are we debugging this still? I see an `stdout` print below.

Reply:
Thanks for catching that, removed in https://github.com/Ekumen-OS/beluga/pull/328/commits/ce9271d9d883bb8c25e64bdb0b24a1aed1fb229e.